.github/workflows/ousd.yml

```diff
@@ -102,7 +102,9 @@ jobs:
 
   slither:
     name: "Slither"
-    runs-on: ubuntu-latest
+    # As long as we need Python 3.6 here in the test, we can only use up to Ubuntu 20.
```

Comment:
Thanks for updating this. We've a low-priority issue to replace Slither with dependabot in future, until then this should work